### PR TITLE
[vscode] tolerate mistyped keybindings

### LINKED ISSUE #4175: [vscode] handle keybindings in upper-case
VS Code normalizes keybindings to lower-case and trim them before parsing. We should do the same.

screen shot 2019-01-28 at 13 34 02

### PR DESCRIPTION
**Pull request by akosyakov**
fix #4175

### CODE REVIEW

File: packages/core/src/browser/keybinding.spec.ts [hunk 1]
```diff
@@ -157,10 +157,10 @@ describe('keybindings', () => {
 
         const bindings = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND2.id);
         if (bindings) {
-            expect(bindings.length).to.be.equal(2);
```
Comment (akosyakov) [inline packages/core/src/browser/keybinding.spec.ts]: @marechal-p @svenefftinge Could you help me to figure out why this test is failing? I cannot get what is testing.

Comment (marechal-p) [reply packages/core/src/browser/keybinding.spec.ts]: I don't understand either... I think before, registering something in the user scope would shadow anything in the default scope. But looking at the way it was before your change, I am confused.

**Comment by marechal-p** [comment]
Seems like there is a test failure on OSX:

https://travis-ci.org/theia-ide/theia/jobs/486813614#L2180

**Comment by akosyakov** [comment]
@marechal-p it is green now, is it good to merge?

File: packages/core/src/browser/keybinding.spec.ts (hunk 1 above)
Comment (svenefftinge) [reply packages/core/src/browser/keybinding.spec.ts]: Same here, the test doesn't make sense to me

**Pull request merged**